### PR TITLE
149 fix missing resistances

### PR DESCRIPTION
**Pull request by samuell**
# Description

Fixes missing OXA-48 and some missing VIM beta-lactamase resistance genes.

## Primary function of PR
- [x] Hotfix
- [ ] Patch
- [ ] Minor functionality improvement
- [ ] New type of analysis
- [ ] Backward-breaking functionality improvement
- [ ] This change requires internal documents to be updated
- [ ] This change requires another repository to be updated

# Testing
_If the update is a hotfix, it is sufficient to rely on the development testing along with the Travis self-test automatically applied to the PR._

_Test routine to verify the stability of the PR:_
- _`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-microsalt-stage.sh BRANCHNAME`_
- _`us`_
- _`source activate S_microSALT`_
- _(SITUATIONAL) `export MICROSALT_CONFIG=/home/proj/dropbox/microSALT.json`_
- _Select a relevant subset of the following:_
- _`microSALT analyse project MIC3109`_
- _`microSALT analyse project MIC4107`_
- _`microSALT analyse project MIC4109`_
- _`microSALT analyse project ACC5551`_

_Verify that the results for projects MIC3109, MIC4107, MIC4109 & ACC5551 are consistent with the results attached to AMSystem doc 1490, Microbial_WGS.xlsx_

## Test results
_These are the results of the tests, and necessary conclusions, that prove the stability of the PR._

# Sign-offs
- [ ] Code tested by @octocat
- [ ] Approved to run at Clinical-Genomics by @sylvinite

### CODE REVIEW

File: tests/test_scraper.py [hunk 1]
```diff
@@ -52,9 +52,13 @@ def test_blast_scraping(scraper, testdata_prefix, caplog):
   caplog.set_level(logging.DEBUG)
   scraper.scrape_blast(type='seq_type',file_list=["{}/blast_single_loci.txt".format(testdata_prefix)])
   assert "candidate" in caplog.text
+
   caplog.clear()
-  scraper.scrape_blast(type='resistance',file_list=["{}/blast_single_resistance.txt".format(testdata_prefix)])
-  assert "candidate" in caplog.text
+  hits = scraper.scrape_blast(type='resistance',file_list=["{}/blast_single_resistance.txt".format(testdata_prefix)])
```
Comment (karlnyr) [inline tests/test_scraper.py:57]:
```suggestion
  hits = scraper.scrape_blast(type='resistance',file_list=[f"{testdata_prefix}/blast_single_resistance.txt"])
```

File: tests/testdata/blast_single_resistance.txt
Comment (karlnyr) [inline tests/testdata/blast_single_resistance.txt:1]: Is there any chance this test file could be toned down to only contain the information you would like to test?

**Review by karlnyr — APPROVED** [review]
Minor comments, nice 👍

File: tests/test_scraper.py (hunk 1 above)
Comment (samuell) [reply tests/test_scraper.py:57]: I wonder if we should perhaps keep the formatting code as is for consistency, and change everything at once later if needed instead?

File: tests/testdata/blast_single_resistance.txt
Comment (samuell) [reply tests/testdata/blast_single_resistance.txt:1]: Yes, I'm happy to remove the VIM-related stuff.

Comment (samuell) [reply tests/testdata/blast_single_resistance.txt:1]: I just dropped all commits related to VIM.

**Comment by samuell** [comment]
Important clarification: This issue did not solve the reported missing VIM genes.

Initially we thought the fix would address both issues, but the VIM issue turned out to have different root cause, and was created as a separate issue #180

**Comment by samuell** [comment]
As can be seen here, `OXA-48` turns up in the Typing report when run on MicroSALT 4.0.0, but did not before for the same sample, where instead `OXA-566` showed up, for the same sample (Ending with `A10`), for case `elegantflamingo`.

MicroSALT 3.5.5, where you can see only `OXA-566` being wrongly found:

image

MicroSALT 4.0.0, where you can see `OXA-48` being correctly found:

image

**Pull request merged**